.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - uses: pre-commit/action@v2.0.0
```

Comment:
can you please summarize what this does?
Also, should we refer to the same pre-commit.com website to understand these meanings?

Reply:
I think we can create another markdown for "GitHub workflow" like we have for pre-commit and explain this there. Let me check it in.